src/helpers/constants/error.messages.js

```diff
@@ -30,6 +30,7 @@ export default {
   ratingsNotFound: 'No ratings found for this article',
   notOwner: 'Please you must be the owner of this Article in order to modify it, Thanks',
   noArticles: 'Article not found!',
-  noMoreArticle: 'There no more articles at the moment.',
+  noMoreArticle: 'There is no more articles at the moment.',
```

Comment:
``There are no more articles at the moment``

Reply:
I resolved this